go/vt/vtorc/logic/tablet_discovery.go

```diff
@@ -55,23 +55,25 @@ var (
 	// This is populated by parsing `--clusters_to_watch` flag.
 	shardsToWatch map[string][]*topodatapb.KeyRange
 
+	// ErrNoPrimaryTablet is a fixed error message.
+	ErrNoPrimaryTablet = errors.New("no primary tablet found")
+)
+
+func init() {
 	// tablet stats
-	statsTabletsWatchedByCell = stats.NewGaugesFuncWithMultiLabels(
```

Comment:
Is it intentional to no longer have this variables being set to the relevant gauges?

Reply:
@ejortegau yes, these stats are set purely from the func provided to `stats.NewGaugesFuncWithMultiLabels`

Setting the `var` is not needed